server/database-postgres/helpers.js

```diff
@@ -0,0 +1,97 @@
+const axios = require('axios');
+const { db, User, Language, UserLanguage } = require('./models.js');
+
+const getUserById = async (userId) => {
+  try {
+    const user = await User.findOne({ where: { userId } });
+    // console.log('😂 getUserById function result', user.dataValues);
+    return user;
+  } catch (err) {
+    console.error(err);
+    return null;
+  }
+};
+
+const getUserLanguagesByUserId = async (userId) => {
+  try {
+    const languages = await UserLanguage.findAll({ where: { userUserId: userId }})
+      .then(ids => ids.map(id => id.languageLanguageId))
+      .then(languageIds => Language.findAll({ where: {languageId: languageIds}}))
+      .then(languageNames => languageNames.map(lang => lang.name))
+      .catch(error => console.log(error));
```

Comment:
does this work? `await` should wait for the promise returned from `findAll` to resolve and store the value to languages.. which you return at the end of `try` block.  It also doesn't look like you are returning anything from the parent function within the chained `then` statements.  it may not throw an error, but to me it looks like this code may not be getting used.  could be wrong!

Reply:
it works. but still not very clear on the whole thing. I tried doing look up using the "include" on the job table but kept getting an error saying userUserId doesn't exist or something. so this was a super janky way to get the data. will have to refactor for sure lol.